Keas.Mvc/Controllers/SuperController.cs

```diff
@@ -1,17 +1,18 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
 
 namespace Keas.Mvc.Controllers
 {
+    [Authorize]
     public class SuperController : Controller
     {
         public string Team => ControllerContext.RouteData.Values["teamName"] as string;
 
         private const string TempDataMessageKey = "Message";
 
+        public string CurrentUserId => User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
```

Comment:
@jcubbage Did you test this?

Reply:
I did....it's not very useful. I think it give you the IAM ID. I forgot to back it out. I've submitted another commit to reverse. The trick is Scott's login sequence takes the email address passed from authentication and stores it as claim. I use it to get user in the security service. Wasn't working because the API wasn't passing credentials.

Reply:
@jcubbage Note that (1) I store a bunch of claims, so other ones might be helpful, but more importantly (2) we'll probably change auth to go back to CAS which means the claims will change a bit anyway.  you are probably better off, if you need something like this, to use the UserManager class to get the current user